Library/Homebrew/cmd/bottle.rb

```diff
@@ -192,10 +204,24 @@ def bottle_formula(f)
 
         keg.delete_pyc_files!
 
+        tab = Tab.for_keg(keg)
+        empty_tab = Tab.empty
+        tab["poured_from_bottle"] = empty_tab["poured_from_bottle"]
```

Comment:
> If you rebottle a poured bottle 

I don't think you could do this? For example, files in `etc` and `var` will be lost.